### ISSUE TITLE
Experimental identification of target for Series 3

### ISSUE THREAD

**Issue by holeung** [OpenSourceMalaria/OSM_To_Do_List#561]
I received a small grant to support experimental, biochemical testing of potential targets for Series 3 (#549) through partnerships with the wonderful core labs (NMR, mass spectrometry, X-ray crystallography, protein expression) of the University of Kansas.

Current approaches that we discussed include:
1. Recombinant expression of candidate kinase/kinase-like targets as discussed in #549. We would prefer to start with those with published expression protocols and that can be expressed in E. coli. We would test for inhibitor binding by saturation transfer difference NMR. We would start by testing our top 5 candidates.

2. Screening of inhibitors against fractions of Plasmodium lysate or enriched lysate, preferably from a non-infectious Plasmodium species or sterilized P. falciparum. We would screen by STD NMR, affinity mass spectrometry, or by something like DARTS ([https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4442491/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4442491)). Anyone know of a source of Plasmodium lysate?

3. Your idea here! Note that target identification is considered difficult and non-routine.

I will keep this sub-project open and under the umbrella and spirit of OSM.

**Comment by drc007** [comment]
Copied from #549 
Since the sulphonamide appears to be interacting with Arg268 perhaps the simplest change to try is to replace the sulphonamide with a carboxylic acid?

A related analogue, https://www.ebi.ac.uk/chembl/compound/inspect/CHEMBL3759356 is known. https://doi.org/10.1016/j.ejmech.2015.11.012 may include useful chemistry.

**Comment by drc007** [comment]
This paper https://doi.org/10.1046/j.1432-1327.2001.02403.x refers to the H89 cAMP-dependent protein kinase inhibitor blocks Plasmodium falciparum development in infected erythrocytes. H89 (https://www.ebi.ac.uk/chembl/compound/inspect/CHEMBL104264) has been extensively studied and might be worth looking at as an additional chemotype.

**Comment by drc007** [comment]
Since this is a kinase target is it worth approaching the ICR (https://www.icr.ac.uk) to see it they have a box of a range of kinase inhibitors that they may be willing to let you test? I'd be happy to make introductions.

**Comment by MFernflower** [comment]
@drc007 rather than screen a whole box - why don't we as a group pick-out interesting already on the market kinase inhibitors? I vote we screen some imatinib - could also ship some of that off to Dundee for potency evaluations

**Comment by drc007** [comment]
I suspect ICR will have access to most/all published kinase inhibitors, together with a variety of molecules that illustrate the different hinge binding motifs.

**Comment by MFernflower** [comment]
@drc007 Fair enough - but screening 20+ kinase inhibitors is a pretty massive task

**Comment by holeung** [comment]
Thanks for the suggestions! Sorry, I may be missing a step here. What is the rationale for screening kinase inhibitors as part of this project? Other groups have already screened some kinase inhibitors against Pf, such as in the Dundee group, Hallyburton (2017), Malar J 16:446, DOI 10.1186/s12936-017-2085-4.

**Comment by MFernflower** [comment]
I think its worth finding out the kinase our s3 drugs hit as opposed to shotgun screening random cancer meds @drc007 @holeung But I did find something interesting: https://clinicaltrials.gov/ct2/show/NCT02614404

**Comment by drc007** [comment]
@holeung I thought you were planning to express potential kinase targets? Almost all kinase inhibitors bind to the hinge binding region, so a simple approach would be test representative examples of different hinge binding motifs to select the most active for your candidate kinase.
@MFernflower Kinase screening is a pretty straightforward assay. 
@holeung If you plan to use saturation transfer difference NMR then will you be looking for fragments? I could give you a list of known kinase inhibiting fragments that you could try.

**Comment by MFernflower** [comment]
I still do not understand why we should screen stuff other than our s3 lead
molecule @drc007

**Comment by drc007** [comment]
@MFernflower Several reasons. Expressing a potential kinase target is a considerable amount of effort, if you only test the s3 lead and it is inactive do you then just drop that potential target? Surely better to have a range of different chemotypes that can perhaps identify an inhibitor that can then be used to evaluate against Plasmodium to see if that kinase is a potential useful target.
Even if s3 is active how do you tell that is the mechanism by which it is acting against Plasmodium? again if you have several different chemotypes (both active and inactive) you can then see if you get the same rank order against Plasmodium.

**Comment by MFernflower** [comment]
@drc007 hence why cell lysate is better than single gene expression!
We screen s3 lead in cell lysate - find what stuff it hits - find compounds
that do the same and screen those on live malaria cells

**Comment by drc007** [comment]
@MFernflower How do you find out what the compound binds to in a lysate?

**Comment by MFernflower** [comment]
that is out of my ability (i only really do lead and synth optimization) -
@holeng

**Comment by MFernflower** [comment]
@holeung perhaps can awnser that - that is out of my knowlage

**Comment by mbhebhe** [comment]
@holeung , that is fantastic news! More people working on Series 3!
We sent OSM-S-106 to UCSD (Elizabeth Winzeler's lab) last year for MoA studies #524. Hopefully both ways will help us get the solution quickly. I have a few mg of OSM-S-106 left but I could make some more if you need a lot. 
@drc007 , the sulfonamide has not been replaced with a carboxylic acid yet. We have replaced with amides and other sulfonamides but not with a carboxylic acid. It looks easy to make, if it works.
![image](https://user-images.githubusercontent.com/26862286/37070631-8517de9e-220c-11e8-8745-784cd0ec1f02.png)

**Comment by mattodd** [comment]
Hi @holeung - it's fantastic that you've secured a grant for this. Congratulations. MoA on Series 3 is super interesting. If the mechanism is new, this molecules is seriously attractive.

ICR for samples to potentially hit the expressed target? Sounds like an excellent idea, particularly if they could (eventually) engage in the relevant conversations here, rather than by email. Very good if you're willing to make intros @drc007

I fully agree that S3 compounds (active and inactive) should be tested alongside other potential inhibitors, since that provides a richer dataset. @MFernflower - adding compounds to lysate works if you've tagged the S3 compounds with something that allows you to fish them out again - usually biotin - a whole extra level of complexity. Here we're being kick-started by @holeung's predictions and those of Vito (#503), giving us rational, specific targets. However, @holeung mentions DARTS, which is a possibility for the lysate idea and which could be interrogated by OSM-S-106 and some negative controls. (By "plasmodium lysate" what do you mean, exactly @holeung - is there a protocol or technical description (I've a pretty good idea of what it means colloquially) that we could circulate to the community?)

However, feeding into this discussion is that the Winzeler lab is currently looking at the Series 3 mechanism of action through the generation of mutants resistant to OSM-S-106 (see #524) as @mbhebhe reminds us above. There are also the MoA-relevant metabolomics data derived from Anubhav Srivastava and Darren Creek, also in #524. These are parallel approaches to those described here, but could strongly inform the selection of specific targets for expression (i.e. bolster approach 1, above). It seems to me that the predictions have given some good, specific targets to look into unless I'm misreading the data.

Synthesis of the carboxylic acid looks like a great target for @mbhebhe and one she already has a plan for, I see above.

Compounds in 10.1016/j.ejmech.2015.11.012 are certainly interesting @drc007. Have we reached out to this group for possible samples - my memory tells me we were thinking of doing that? I'm happy to unless someone else knows them.

**Comment by drc007** [comment]
@mattodd @holeung I'm happy to contact ICR and authors of 10.1016/j.ejmech.2015.11.012.

**Comment by mattodd** [comment]
One of the authors of 10.1016/j.ejmech.2015.11.012 has asked by email "How much material is needed and in what form (solid, dissolved etc?)" @holeung I guess it depends what we're doing with them, right? But 2 mg solid form would be enough for most things, I'd imagine?

**Comment by MFernflower** [comment]
We should send the compounds out to dundee for potency eval too @mattodd

**Comment by drc007** [comment]
Would it worth contacting the authors here ? https://doi.org/10.1371/journal.pone.0181585

**Comment by holeung** [comment]
Thanks so much for all the ideas and reaching out to other groups. This is a small, one year pilot grant which will hopefully lead to bigger things... exploration of the plasmodium kinome and full structure-based drug design of OSM compounds. I am busy with the paperwork but will be updating my ELNs here on the docking we've done on the full list of essential kinases. We will have limited screening ability (well, we have the ability but probably not the $$$), here but hopefully can share protein with other labs with more HTS abilities.

**Comment by holeung** [comment]
Yes, 2 mg of compounds should be plenty. Can we ask for 5-10 mgs of OSM-S-106?

We haven't been able to find a source of plasmodium lysate so we will have to put those experiments on hold. Lysate would be isolated plasmodium from erythocytes, then broken up in a fancy blender, and then fractionated into membrane and non-membrane associated components.

**Comment by mbhebhe** [comment]
We can give you 5 mg of OSM-S-106

**Comment by drc007** [comment]
@holeung @mattodd I think exploration of the plasmodium kinome could be very fruitful. I'd encourage you to aim big! A collaboration with the efforts to produce a  public chemogenomic set for protein kinases (http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0181585) could be a really useful way forward. This could give you chemical starting points for any kinases that you isolate without the need to resort to HTS.

**Comment by mattodd** [comment]
Agreed @drc007 - a great team to work with in [that paper](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0181585). In the short term (i.e. here) we need to validate that we are dealing with a kinase inhibitor in OSM-S-106, though, right? Longer term we could request probes of this kind against kinases expressed by @holeung (generating new OSM series) and obviously contribute OSM starting points to any [ongoing kinome projects](https://www.ncbi.nlm.nih.gov/pubmed/25111632). Did you have something else in mind for now @drc007 ?

@holeung shall we ship OSM-S-106 to you now, or later?

**Comment by drc007** [comment]
@mattodd In the first instance validation work around OSM-106 is top priority.

The work described in the paper is targeted towards human kinases, I was wondering if we should see if they might be interested in collaborating in developing a kinase inhibitor set that might be directed towards non-human targets. Perhaps DNDI and MMV might be interested?

**Comment by holeung** [comment]
Thanks, everyone, especially @drc007 for encouraging me to think big. 5 mgs of OSM-S-106 should be plenty. We won't be ready to start experiments on the compound for a month or so. I'll send shipping instructions later. In the meantime, I have quite a bit of formal paperwork to prepare on my side.

**Comment by mattodd** [comment]
@drc007 I guess I'm surprised there is not already a malaria kinase box.
Interesting obviously in part because different inhibitors may be
differently effective against different life cycle stages. i.e. a rich
source of data. Sounds like a great project in which many will be
interested. Making me wonder if it's already being looked at?

On 21 Mar 2018 11:30 pm, "Chris Swain" <notifications@github.com> wrote:

@mattodd <https://github.com/mattodd> In the first instance validation work
around OSM-106 is top priority.

The work described in the paper is targeted towards human kinases, I was
wondering if we should see if they might be interested in collaborating in
developing a kinase inhibitor set that might be directed towards non-human
targets. Perhaps DNDI and MMV might be interested?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/561#issuecomment-374920682>,
or mute the thread

**Comment by drc007** [comment]
I've asked around and everyone seems surprised there is no kinase box for neglected and tropical diseases. Perhaps worth looking at creating?

Apparently GSK Tres Cantos has a 56 P. falciparum kinase panel, against which the S3 compound could be screened.

**Comment by mattodd** [comment]
Nice. I just pinged Mike Pollastri by email to ask him about this. And unless you say nay I'll reach out to the GSK Tres Cantos guys to see if such a screen of Series 3's OSM-S-106 (and negative control) would be acceptable.

**Comment by drc007** [comment]
@mattodd all fine with me

**Comment by mpollastri** [comment]
@drc007 @mattodd @holeung 

 I'm not aware of a parasite kinase box akin to the KinetoBox or MalariaBox.....however, I do think that GSK was at one point willing to plate out the cluster representatives from our[ kinase targeted library screening paper](http://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0003253) against T brucei. Note however that these were only tested against T brucei (to my knowledge).... The [PKIS set](https://pharmacy.unc.edu/research/sgc-unc/) might be another option, if you've not already considered it. 

I'm happy to contact GSK to inquire, if you wish.

**Comment by holeung** [comment]
The more compounds we can get out to the public, the better! Thanks, @mpollastri .

**Comment by mattodd** [comment]
Hi @mpollastri - that'd be very good, yes. a) OSM would love to test OSM-S-106 against any relevant kinases (e.g. if GSK ran such assays) and b) the idea of a box of compounds with proven ability to inhibit kinases relevant to NTDs might be an interesting idea. I certainly think it's interesting, as do other people here, but I'd be super-interested if GSK had considered this and either want to pursue it or decided against it for some reason. If you're happy to reach out to them and report back (to the extent they are happy for that) that'd be very good. 
Oh and welcome aboard, Mike. We're lucky to have you contribute directly.

**Comment by drc007** [comment]
@mattodd @mpollastri Great if we can screen against Tres Cantos kinase screen, could help is focus efforts quickly.

Whilst it is useful to have a set of kinase inhibitors with proven activity against NTD targets, I'd prefer to include to include examples of as many chemotypes as possible that have shown to be active against kinases, irrespective of the kinase target.

**Comment by PaulWillisMMV** [comment]
Are you aware of the human kinase inhibitors box `The Published Kinase Inhibitor Set (PKIS)`- (many of which, I’m sure would impact pathogens) available for screening from GSK - link here + contact details for obtaining the box. 

https://www.ebi.ac.uk/chembldb/extra/PKIS/

The Published Kinase Inhibitor Set (PKIS) is a collection of 376 compounds that have been made available by GSK for screening by external groups; all compounds have been published in the scientific literature. 

MMV can provide aditional contact info, if interested DM me

MMv has not therefore prioritised the assembly of a kinase box  We do have 3 new boxes in planning, details soon

**Comment by drc007** [comment]
Hi,

It was actually reading about the human kinase inhibitor box that initiated this idea. If you can provide contacts it would be appreciated 

Cheers,

Chris

**Comment by PaulWillisMMV** [comment]
Bill Zuercher is now at UNC but still distributes the compounds.  His contact details are william.zuercher@unc.edu

If you are planning an NTD kinase box it may be worth thinking about how groups obtaining new hits could be further supported to optimise these hits to obtain a desired selectivity (particularly over human kinases) 

Paul

**Comment by mpollastri** [comment]
Mat – sorry for the delay.

I will write to GSK and ask about construction of a box. I am not sure if they’d run a kinase panel. I’ll check.

**Comment by MFernflower** [comment]
Do not really know if this is of help to you guys but it seems human TYKI's can kill malaria parasites: https://www.ncbi.nlm.nih.gov/pubmed/26142327

**Comment by bendndi** [comment]
Hi all,
Just had some free time to pop in and see what was going on in OSM. Thought I'd add in some design ideas based on this putative docking:
![image](https://user-images.githubusercontent.com/36627399/41917504-da508a78-795a-11e8-9f71-d89ea93b792b.png)

**Comment by drc007** [comment]
Also comments on twitter https://twitter.com/O_S_M/status/1011477937806237696

Aryl sulphonamides are known carbonic anhydrase inhibitors.

**Comment by MFernflower** [comment]
@bendndi Sorry it's been so long but do you think you could dock the diamino variant of the lead compound eg:


NC=1C2=C(N=C(N1)N)C=C(S2)C=2C=C(C=CC2)S(=O)(=O)N

![3- 4 2diaminothieno 3 2-d pyrimidin-6-yl benzenesulfonamide](https://user-images.githubusercontent.com/3164942/41932120-224852be-794e-11e8-8cfe-c14f4e747581.png)

**Comment by MFernflower** [comment]
@drc007 Would be neat to screen topamax and acetazolamide against malaria!

**Comment by drc007** [comment]
@MFernflower Topamax is a promiscuous drug with some serious adverse effects, I'm not sure it would be a useful starting point.

Acetazolamide could well have been tested since it is most screening collections. 

Sulphonamides can cause allergic reactions (https://www.ncbi.nlm.nih.gov/pubmed/17504660) but I don't know enough about the mechanism to say which might be a concern.

**Comment by MFernflower** [comment]
@drc007 pertiant paper
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3609187/#!po=45.7746

Might be worth sending some ace over to Dundee for our own potency eval?

**Comment by drc007** [comment]
@MFernflower Indeed, and there are a number of more recent publications suggesting CA as a good target. This together with the extensive availability of known CA inhibitors raises the question what is the stumbling block. @PaulWillisMMV @mattodd  is there a particular problem with this approach?

**Comment by MFernflower** [comment]
@mattodd would it be possible for you to phone up Dundee so you can set in
motion the screening of Dorzolamide?
Would be nice to get a base line potency of a pure CA hitter vs the s3 hit

**Comment by mpollastri** [comment]
The primary issue with CA is one of selectivity versus host. It’s tricky sometimes. Not that this compound should be avoided or dropped, but go into it with eyes open!

**Comment by MFernflower** [comment]
@drc007 Do you think it would be worth making this just to see how much of a role CA blockade plays in the activity of the S3 drugs? 

![2w22](https://user-images.githubusercontent.com/3164942/41945874-703d7920-797d-11e8-8884-63cc653f7a6d.png)

**Comment by drc007** [comment]
Given the above discussion I've been looking at the series 3 compounds.
OSM-S- (IC50 0.2 uM) is a puzzle. According to the structure in the spreadsheet shown below, this does not contain a sulphone. I suspect that this structure is incorrect, can we check?

![osm-s](https://user-images.githubusercontent.com/7080516/41956574-01a38464-79dc-11e8-927f-db39cfc3696f.png)

**Comment by MFernflower** [comment]
If that structure is accurate and is really able to kill at 200 nanomolar - what on earth could it be targeting? It looks super strange @drc007

**Comment by MFernflower** [comment]
Also I feel we aught to at some point spin off the CA inhibition to a issue in the s3 repo - things are getting super tangled especially with the s3 hit

**Comment by mbhebhe** [comment]
Its actually [TCMDC-134392](https://pubchem.ncbi.nlm.nih.gov/substance/85863501#section=Top). Its a GSK compound and it doesn't have an OSM number. But it does have an activity value of 0.23 uM

**Comment by david1597** [comment]
It has been assigned OSM-S-590.

**Comment by drc007** [comment]
Sure it is not http://www.chemspider.com/Chemical-Structure.24574288.html

**Comment by holeung** [comment]
Yes, I would also be interested to know if the Series 3 compounds block carbonic anhydrase.

**Comment by drc007** [comment]
I've written a comparison of several bioactivity prediction packages. 
https://www.macinchem.org/reviews/bioactivities/bioactivities.php

I included OSM-S-106 as a test compound, unsurprisingly Carbonic Anhydrase inhibition and a variety of kinase targets are flagged.

**Comment by MFernflower** [comment]
@drc007 here is another free service that could be worth testing
http://www.swisstargetprediction.ch/

**Comment by drc007** [comment]
@MFernflower That uses pretty old version of ChEMBL (v16) and I don't think it as been updated since 2103?

**Comment by MFernflower** [comment]
@drc007 that could explain why it sometimes returns screwy results
 PBB2 seems much more reliable
